tests/test_optional.py

```diff
@@ -44,108 +43,109 @@ def test_can_get_when_present_and_have_checked(self):
         assert optional.get() == "thing"
 
     def test_will_run_consumer_if_present(self):
-        scope = {'seen': False}
+        scope = {"seen": False}
 
         def some_thing_consumer(thing):
-            scope['seen'] = True
+            scope["seen"] = True
 
         optional = Optional.of("thing")
         optional.if_present(some_thing_consumer)
-        assert scope['seen']
+        assert scope["seen"]
 
     def test_will_not_run_consumer_if_not_present(self):
-        scope = {'seen': False}
+        scope = {"seen": False}
 
         def some_thing_consumer(thing):
-            scope['seen'] = True
+            scope["seen"] = True
 
         optional = Optional.empty()
         optional.if_present(some_thing_consumer)
-        assert not scope['seen']
+        assert not scope["seen"]
 
     def test_will_run_or_else_from_if_present_when_not_present(self):
         scope = {
-            'if_seen': False,
-            'else_seen': False,
+            "if_seen": False,
+            "else_seen": False,
         }
 
         def some_thing_consumer(thing):
-            scope['if_seen'] = True
+            scope["if_seen"] = True
 
         def or_else_procedure():
-            scope['else_seen'] = True
+            scope["else_seen"] = True
 
         optional = Optional.empty()
         optional.if_present(some_thing_consumer).or_else(or_else_procedure)
-        assert not scope['if_seen']
-        assert scope['else_seen']
+        assert not scope["if_seen"]
+        assert scope["else_seen"]
 
     def test_will_not_run_or_else_from_if_present_when_not_empty(self):
         scope = {
-            'if_seen': False,
-            'else_seen': False,
+            "if_seen": False,
+            "else_seen": False,
         }
 
         def some_thing_consumer(thing):
-            scope['if_seen'] = True
+            scope["if_seen"] = True
 
         def or_else_procedure():
-            scope['else_seen'] = True
+            scope["else_seen"] = True
 
         optional = Optional.of(23)
-        value = optional.if_present(some_thing_consumer).or_else(or_else_procedure)
-        assert scope['if_seen']
-        assert not scope['else_seen']
+        value = optional.if_present(some_thing_consumer).or_else(
```

Comment:
I assume we did a line break here because of the new 80 character limit?  My preference would be:
```python
value = \
    optional.if_present(some_thing_consumer) \
        .or_else(or_else_procedure)
```
This formatting ensures that the readers eye always sees the indented level as directly (or as closely as possible) subbed to the item it relates.  

Off this explanation I also get people on my team doing things like:
```python
value = \
    optional \
        .if_present(some_thing_consumer) \
        .or_else(or_else_procedure)
```
I dont mind this, as I think it reads really easy, but something about that lone `optional \` is a bit strange.

Thoughts?

Reply:
I wonder if black can be configured like this. It presents itself as an 'opinionated' formatter.

Reply:
¯\_(ツ)_/¯

I could pull and try and figure it out but I am pretty impartial on linters.  I can live with the above formatting if its not configurable.  Curious about your preferences on the above styles though.